projects/demo/src/modules/components/text-area/examples/4/index.html

```diff
@@ -35,6 +35,9 @@
         >
             Type it
         </tui-text-area>
-        <tui-field-error formControlName="testValue1"></tui-field-error>
+        <tui-error
+            formControlName="testValue1"
+            [error]="[] | tuiFieldError | async"
```

Comment:
It's curious why the behaviour was changed 🤔 

<img src="https://user-images.githubusercontent.com/35179038/173767758-66b686c0-99b3-405d-b353-e69f2d1f3990.png" height="400">

Reply:
anyway, it not a problem of current PR.
I will create an issue for this problem.